.env.example

```
@@ -10,3 +10,4 @@ MAILJET_USER="123456789abcdef123456789abcdefgh"  #given on MJ account
 MAILJET_PASSWD="123456789abcdef123456789abcdefgh" #given on MJ account
 MAILJET_SENDER="leximpact@openfisca.org" #must be validated on mailjet the first time it's used.
 DATA_PATH="dummy_data.h5" #will look for a local file if ends with .h5 or .csv, else will call the table named this way in the database
+NAME_TABLE_BASE_RESULT = "base_results"
```

Retour à la ligne manquant :) :
```suggestion
NAME_TABLE_BASE_RESULT = "base_results"

```